packages/fxa-auth-server/fxa-oauth-server/config/dev.json

```diff
@@ -176,13 +177,31 @@
       "privacyUri": "",
       "trusted": true,
       "publicClient": false
+    },
+    {
+      "id": "a8c528140153d1c6",
```

Comment:
Can probably remove Secure Proxy from config and only add it to the config needed on stomlinson.dev.lcip.org.